src/abstractdatatable/io.jl

```diff
@@ -45,7 +45,7 @@ function printtable(io::IO,
             if !isnull(dt[j][i])
                 if ! (etypes[j] <: Real)
                     print(io, quotemark)
-                    x = isa(dt[i, j], Nullable) ? _unsafe_get(dt[i, j]) : dt[i, j]
+                    x = isa(dt[i, j], Nullable) ? NullableArrays.unsafe_get(dt[i, j]) : dt[i, j]
```

Comment:
I'd rather not rely on NullableArrays internals for this. Also in Julia 0.6 `unsafe_get exists in Julia Base, so better not make it look like a NullableArrays-specific function.